Code/Framework/AzCore/AzCore/Math/MathUtils.h

```diff
@@ -468,16 +470,24 @@ namespace AZ
     constexpr uint32_t DivideAndRoundUp(uint32_t value, uint32_t alignment)
     {
         AZ_Assert(alignment != 0, "0 is an invalid multiple to round to.");
-        return ((value + alignment - 1) / alignment);
+        AZ_Assert(
+            AZStd::numeric_limits<uint32_t>::max() - value >= alignment,
+            "value '%" PRIu32 "' and alignment '%" PRIu32 "' will overflow when added together during DivideAndRoundUp.");
```

Comment:
You forgot to include arguments for value and alignment

Reply:
Great catch, thanks! Fixed.